### ISSUE TITLE
Test case: Condition

### ISSUE THREAD

**Issue by jkiddo** [Gravitate-Health/focusing-interfaces#5]
Given the drug of http://build.fhir.org/ig/HL7/vulcan-eproduct-info/Bundle-bundlepackageleaflet75.html and the IPS of https://build.fhir.org/ig/hl7-eu/gravitate-health-ips/Bundle-gravitate-helen.json.html (Condition with ID de06df6b-0967-4b01-822b-7cfeea970aa7) - GLens focus should spot the diabetes alert (ClinicalUseDefinition with ID 83f4a22f-9e5e-4941-9fdb-339a0f1f2055) and highlight that in the returned ePI.

**Comment by jkiddo** [comment]
FYI @cander2

**Comment by cander2** [comment]
Yes, it should. Will have to get Martin to confirm this but Humalog is indicated to treat diabetes and Karvea is indicated to treat hypertension but it has a contraindication for those with renal impairment or low blood sugar. So, G-Lens should be able to flag these two.
